pkg/idtools/idtools_supported.go

```diff
@@ -17,6 +17,14 @@ struct subid_range get_range(struct subid_range *ranges, int i)
 {
     return ranges[i];
 }
+
+// get_sub[ug]id_ranges were renamed to subid_get_[ug]id_ranges in
+// libsubid version 4.
+#if SUBID_ABI_MAJOR >= 4
```

Comment:
I think it would be better to go the other way and use the new names in code and then alias the old names with the new names.

Reply:
fixed